### DISCUSSION TITLE
Brak informacji o liczbie sztuk produktów w ofercie publicznej (zestaw produktowy)

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#8572] [Suggestions - rejected]
Chodzi o dane zwracane z zasobu **/offers/listing**.
Myślę, że dla spójności i zachowania rzetelności zwracanych danych informacja o liczbie produktów wchodzących w skład zestawu, którego cena dotyczy, powinna być również przekazana.

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ aktualnie zasobu GET /offers/listing nie ma w publicznym API.
